stx-genesis/src/lib.rs

```diff
@@ -81,6 +86,44 @@ impl GenesisData {
             include_bytes!(concat!(env!("OUT_DIR"), "/names.gz"))
         })
     }
+    pub fn read_name_zonefiles(&self) -> Box<dyn Iterator<Item = GenesisZonefile>> {
+        read_deflated_zonefiles(if self.use_test_chainstate_data {
+            include_bytes!(concat!(env!("OUT_DIR"), "/name_zonefiles-test.gz"))
+        } else {
+            include_bytes!(concat!(env!("OUT_DIR"), "/name_zonefiles.gz"))
+        })
+    }
+}
+
+struct LinePairReader {
+    val: Lines<BufReader<Decoder<Cursor<&'static [u8]>>>>,
+}
+
+impl Iterator for LinePairReader {
```

Comment:
(For my own sanity) anyone know why `chunks` is implemented for slices and vecs, but not for iterators? This was implemented because `my_iter.chunks(2)` wasn't possible..